index.md

```diff
@@ -224,6 +224,19 @@ modification depends on whether you updated your dependencies in order to fix
 a bug or introduce new functionality. I would usually expect additional code
 for the latter instance, in which case it's obviously a minor level increment.
 
+#### Special note for JVM or similar environments
```

Comment:
The FAQ is currently part of the Semver spec itself, thus this should go in https://github.com/semver/semver/blob/master/semver.md. When the next version of Semver is finalized, it'll get copied over here and translated.

Reply:
There is a PR https://github.com/semver/semver/pull/414 and a comment https://github.com/semver/semver/pull/414#pullrequestreview-80298503

Reply:
@Primetalk, @Haacked is the authority on this, my apologies for any confusion.